modules/apps/27-interchain-accounts/controller/migrations/v6/migrations_test.go

```diff
@@ -17,6 +17,12 @@ import (
 	ibcmock "github.com/cosmos/ibc-go/v8/testing/mock"
 )
 
+var (
+	metadata        = icatypes.NewMetadata(icatypes.Version, ibctesting.FirstConnectionID, ibctesting.FirstConnectionID, "", icatypes.EncodingProtobuf, icatypes.TxTypeSDKMultiMsg)
+	versionBytes, _ = icatypes.ModuleCdc.MarshalJSON(&metadata)
+	version         = string(versionBytes)
+)
+
```

Comment:
Can you move this to inside `SetupTest` since it seems it is the only place it is used?

Reply:
I updated this code

Reply:
This is resolved